src/fides/api/ops/service/masking/strategy/masking_strategy_aes_encrypt.py

```diff
@@ -59,10 +60,14 @@ def mask(
 
             masked_values: List[str] = []
             for value in values:
+                if value is None:
+                    masked_values.append(None)
+                    continue
+
                 nonce: bytes | None = self._generate_nonce(
-                    value, key_hmac, request_id, masking_meta  # type: ignore
+                    str(value), key_hmac, request_id, masking_meta  # type: ignore
```

Comment:
If `value` is a member of `List[str]` why do we need this? Looks like sometimes it's explicitly passed as `None`?

I know we're handling it above explicitly but a common gotchya is that `str(None)` actually yields `"None"` FYI

Reply:
Yes, correct, when executing a DSR, the occurs when if the masking strategy targets a non-string field.

Yep, I didn't want str(None) to be turned in a "None" so I handle that separately.